pkg/drivers/kic/oci/network_create.go

```diff
@@ -36,6 +37,9 @@ const firstSubnetAddr = "192.168.49.0"
 // big enough for a cluster of 254 nodes
 const defaultSubnetMask = 24
 
+// name of the bridge network that docker creates by default to be used to get the MTU. ( related issue #9528)
+const dockerDefaultBridgeName = "bridge"
```

Comment:
See if you can shorten the name and still keep context: `dockerDefaultBridge` or `dockerDefaultNetwork`. The `Name` is  somewhat implicit.